src/css/custom.css

```diff
@@ -90,7 +92,7 @@ p {
 }
 
 .menu>.menu__list .menu__list{
-  border-left: 1px solid #18e888;
+  border-left: 2px solid #18e888;
```

Comment:
we **do not** use statically defined colours, use the theme variables like how is defined on line 114, among others.

Reply:
addressed in #1194